pkg/client/image.go

```diff
@@ -17,16 +18,21 @@ func (c *client) ImageTag(ctx context.Context, imageName, tag string) error {
 	if err != nil {
 		return err
 	}
-
-	tagResult := &apiv1.ImageTag{}
-	err = c.RESTClient.Post().
+	if includesTag(tag) {
+		image.Tags = append(image.Tags, strings.Split(tag, ":")[1])
+	}
+	if image.References[0] == "" {
```

Comment:
I believe something is breaking in the client. When running locally, I can't build as I keep hitting an out of range error referencing this line of code.

```console
$ ./bin/acorn build -t test/test:v0.0.1 .
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/acorn-io/acorn/pkg/client.(*client).ImageTag(0x1400088a500, {0x1058c3038, 0x140006b0e40}, {0x1400094dcc0?, 0x0?}, {0x140004ef570, 0x10})
        /Users/tylerslaton/Code/acorn/pkg/client/image.go:24 +0x3a8
```

For what its worth, this is the Acornfile I'm using.

```cue
containers: {
    nginx: {
        image: "nginx"
        ports: publish: "80/http"
        files: {
            "/usr/share/nginx/html/index.html": "<h1>My first Acorn!</h1>"
        }
    }
}
```

Reply:
Let me know if this is just an issue on my end. I had a fresh install using a binary based on your branch so I'm not sure what would've went wrong otherwise.

Reply:
tried to replicate but seems to be building on my end. I am using this to install acorn tho 
```
alias build='make build && docker build -t joshuasilverio/acorn:img . && docker push joshuasilverio/acorn:img && ./bin/acorn install --image joshua
silverio/acorn:img --controller-replicas=0'
```
and restarting the local controller and restarting the api pod

Reply:
Gotcha, it must be my image isn't getting set in the acorn install for the controller or server or something. I'll resolve this since I believe this is just on my end. I'll nuke the cluster and validate.